boefjes/boefjes/local.py

```diff
@@ -40,10 +42,14 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 class LocalBoefjeJobRunner(BoefjeJobRunner):
     def __init__(self, local_repository: LocalPluginRepository):
         self.local_repository = local_repository
+        self.scheduler_client = SchedulerAPIClient(str(settings.scheduler_api))
 
     def run(self, boefje_meta: BoefjeMeta, environment: Dict[str, str]) -> List[Tuple[set, Union[bytes, str]]]:
         logger.info("Running local boefje plugin")
 
+        task_id = str(boefje_meta.id)
+        self.scheduler_client.patch_task(task_id, TaskStatus.RUNNING)
+
```

Comment:
I don't know if adding this is advisable or desirable. I took note from the docker boefje runner. @Donnype